### ISSUE TITLE
[Caching issue] Region for minerals set to delve intermittently 

### ISSUE THREAD

**Issue by charliedimaggio** [Kyria/LazyBlacksmith#64]
This has happened before and I made a previous post about it. I have more information this time. This is happening even when I go into incognito mode. 
I have linked a screenshot that will show that the minerals have defaulted to Delve. https://i.imgur.com/sC229r1.png
I am not logged in and I am using incognito to avoid potential caching issues. I have never set the region to Delve as I do not have any interest in that region.
Other users have been able to replicate the problem. 
I will ask them to comment on this thread.

**Comment by RaylinnKisiel** [comment]
I can confirm the above information, I replicated what the original poster did, and tried it in and out of incognito mode.
https://imgur.com/a/OG2RhHL

**Comment by Kyria** [comment]
Ok, i didn't understood the issue before. 

Actually, as I just checked, it's not directly a bug, but actually a cache issue, as sub materials are gathered after the page is loaded, and somehow the cache for anonymous user was ... weird. 

I'll have to look at it. Meanwhile, i've cleared the cache, so it "should" be all in the forge now. 

I also changed your issue title, to have the most important informations in it. 
And to remind myself: https://github.com/Kyria/LazyBlacksmith/blob/master/lazyblacksmith/views/ajax/eve_sde.py#L96 has been deleted temporarily. 

Thanks for taking the time to report it :)
